HVZ/HVZ/api/views.py

```diff
@@ -61,7 +71,7 @@ def form_valid(self, form):
         elif(recipient_title == MailerForm.ZOMBIES):
             recipients = [p.user.email for p in Player.current_players() if p.team == "Z"]        
 
```

Comment:
These cases seem a bit redundant if there are more cases. An more efficient way would be better.

Reply:
I agree. Especially if we plan to sort players in groups where there are more than two options. Perhaps we could change the list comprehension condition to read in the mailer form choice.

Reply:
I agree @hmc-cs-ctan and @AdamSchulze . If we look towards our stretch goals and want to separate players by Class or by College, these checks in the code will begin to be much more redundant. We could consider check-boxes for each group/subgroup to choose which groups will receive the email, however we would still need a more clean back-end implementation.